### PR TITLE
update sdk version to 1.19.1

### PR DESCRIPTION
**Pull request by adi88d**
## Status
Ready

## Related Content Pull Request
Related PR: link to the PR at demisto/content

## Related Issues
Related: link to the issue

## Description
A few sentences describing the overall goals of the pull request's commits.

### CODE REVIEW

File: docker/xsoar-tools/Pipfile [hunk 1]
```diff
@@ -5,7 +5,7 @@ name = "pypi"
 
 [packages]
 demisto-py = "*"
-demisto-sdk = "==1.17.2" # this package should be updated manually. No reason this is locked
+demisto-sdk = "*"
```
Comment (JudahSchwartz) [inline docker/xsoar-tools/Pipfile:8]: didnt we want this locked ? @dorschw

Comment (dorschw) [reply docker/xsoar-tools/Pipfile:8]: indeed, it should be locked.

Comment (GuyAfik) [reply docker/xsoar-tools/Pipfile:8]: why is it locked? why can't we update it with depandabot?

Comment (dorschw) [reply docker/xsoar-tools/Pipfile:8]: we can, but want to control it manually.

Comment (JudahSchwartz) [reply docker/xsoar-tools/Pipfile:8]: hence the comment

Comment (GuyAfik) [reply docker/xsoar-tools/Pipfile:8]: @adi88d i still dont understand why we want to control it manually but go ahead and lock it.
i don't see any reason why we need to update it manually each time, it makes some of our content scripts to run with old SDK versions which contain bugs that were fixed in later versions

Comment (dorschw) [reply docker/xsoar-tools/Pipfile:8]: See [here](https://panw-global.slack.com/archives/C03L972AER1/p1692798368468139)

Comment (dorschw) [inline docker/xsoar-tools/Pipfile:8]: change to `1.19.1`

**Review by dorschw — CHANGES_REQUESTED** [review]

File: docker/xsoar-tools/Pipfile [hunk 2]
```diff
@@ -5,7 +5,7 @@ name = "pypi"
 
 [packages]
 demisto-py = "*"
-demisto-sdk = "==1.17.2" # this package should be updated manually. No reason this is locked
+demisto-sdk = "1.19.1" # this should only be modified manually. only specify a version, do not use *.
```
Comment (dorschw) [inline docker/xsoar-tools/Pipfile:8]:
```suggestion
demisto-sdk = "==1.19.1" # this should only be modified manually. only specify a version, do not use *.
```

**Review by dorschw — APPROVED** [review]

**Review by dorschw — APPROVED** [review]

**Pull request merged**